WhisperAPI/WhisperAPI.Tests/Unit/SuggestionsServiceTest.cs

```diff
@@ -104,6 +104,37 @@ public void When_query_is_selected_by_agent_suggestion_is_filter_out()
                 x.Uri == "https://onlinehelp.coveo.com/en/cloud/Coveo_Cloud_Query_Syntax_Reference.htm"));
         }
 
+        [Test]
+        [TestCase("*#")]
+        [TestCase("I*")]
+        [TestCase("*like*")]
+        [TestCase("I*i*C*")]
+        [TestCase("I like C#")]
+        public void When_Intent_With_and_without_Wildcard__is_irrelevant(string wildcardString)
```

Comment:
This is good, but you need to test when the query should be relevant. You are only testing when it should be irrelevant. We need another unit test for this. I'm pretty confident that your code works just fine, but it's to prevent that we break this code later on. Imagine if we make a change that makes all queries irrelevant, then you test would still pass, but we would have a big problem if it were the case.